scripts/ci/xcodebuild_noninteractive.py

```
@@ -177,42 +297,110 @@ def main() -> int:
         ),
     )
 
+    group_receipt_read, group_receipt_write = os.pipe()
     pid, fd = pty.fork()
     if pid == 0:
+        os.close(group_receipt_read)
         try:
             os.setsid()
         except OSError:
             pass
```

Ruff (0.16.1)

[warning] Use `contextlib.suppress(OSError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(OSError): ...`

(SIM105)